konfera/templates/konfera/event_speakers.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'konfera/base.html' %}
+
+{% block title %}<h1>{{ event }} Speakers</h1>{% endblock %}
```

Comment:
Do you happen to have this in event_sponsors.html ?
 `{% block content %} `
 `     <h1>{{ event }}</h1>`